qos/cometbft/routes.go

```diff
@@ -0,0 +1,10 @@
+package cometbft
+
+const (
+	// Get node health. Returns empty result (200 OK) on success, no response - in case of an error.
+	// Reference: https://docs.cometbft.com/v0.38/rpc/#/Info/health
+	routeHealthCheck = "/health"
```

Comment:
is `route` a word used regularly in cometbft docs? If so, please reference it here.
Otherwise, names like `paths.go` or `api_paths.go` may be a better fit for this file.

Reply:
Agreed. I had been trying to avoid too many uses of the word `path` given the uh, name of the repo but `apiPath` is specific enough to avoid confusion.